providers/feature_dism.rb

```diff
@@ -69,3 +67,14 @@ def dism
     locate_sysnative_cmd('dism.exe')
   end
 end
+
+@@feature_list = nil
+def get_feature_list
```

Comment:
Might be nice to have this create a hash of features to avoid the parsing in the consuming methods